src/node_util.cc

```diff
@@ -174,6 +176,15 @@ void PromiseReject(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret.FromMaybe(false));
 }
 
+void SafeGetenvForJs(const FunctionCallbackInfo<Value>& args) {
+  CHECK(args[0]->IsString());
+  Local<String> envtag = args[0].As<String>();
+  String::Utf8Value strenvtag(envtag);
+  std::string text;
+  if (!SafeGetenv(*strenvtag, &text)) return;
+  args.GetReturnValue()
+      .Set(String::NewFromUtf8(args.GetIsolate(), text.c_str()));
```

Comment:
Can you use the String::NewFromUtf8 overload that returns a MaybeLocal?

Apropos the function name, I remember discussing renaming it to just SafeGetenv?

Reply:
done 👍